gateway/app/main/events/fl_events.py

```diff
@@ -0,0 +1,153 @@
+import uuid
+import json
+
+from .socket_handler import SocketHandler
+from ..codes import MSG_FIELD, RESPONSE_MSG, CYCLE
+from ..processes import processes
+
+# Singleton socket handler
+handler = SocketHandler()
```

Comment:
We'd like to have the Grid client callable via Websockets **and** HTTP. This will be up to the client to determine which they would like to use. Just confirming - is this still possible according to this file structure?

Reply:
Yep! This Socket Handler is just to map the socket connections(descriptors) by the worker's ids (we can use this structure to send messages by groups/scopes/rooms). But, is still possible to work with HTTP requests.